templates/write_contributors.py.in

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+#!/usr/bin/env python
+#
+# Generates CONTRIBUTORS.rst file, with the name and email of the contributors
+# to the project, sorted from more to less commits.
+
+import os
+import re
+import subprocess
+import sys
+
+SRC = os.getcwd()
```

Comment:
@xispa @ramonski same here as my previous comment. For some reason `${buildout:directory}` didn't get much love.